crates/turborepo-lib/src/run/scope/filter.rs

```diff
@@ -735,7 +751,7 @@ mod test {
             change_detector,
         );
 
-        resolver
+        (temp_folder, resolver)
```

Comment:
It is *very* important we return the `TempDir` variable as when it gets dropped/out of scope it marks a folder ready for cleanup which leads to non-deterministic tests.

Reply:
put this in a code comment instead?